assemblies/assembly_using-ansible-rhdm-in-ONAP/main.adoc

```diff
@@ -0,0 +1,38 @@
+[id='assembly_using-ansible-rhdm-in-ONAP']
+
+include::_artifacts/document-attributes.adoc[]
+
+= Policy based decision and control in ONAP using Ansible and {PRODUCT}
+// Context attribute is assembly specific and enables module reuse between assemblies.
+:doctype: book
+:imagesdir: _images
+:context: using-ansible-rhdm-in-ONAP
+include::_artifacts/author-group.adoc[]
+
+// Purpose statement for the assembly
+This solution examines the use of Ansible and {PRODUCT} technology in ONAP architecture. It elaborates on how automation and rules management systems together can provide a comprehensive network policy engine capable of providing the necessary automations for proactive response to network and service conditions without human intervention which is also known as closed loop automation.
```

Comment:
The sentence is too long and incomprehensible. Please consider splitting into smaller ones.

Reply:
Changed as suggested.